packages/components-js/projects/partials/tests/unit/specs/__snapshots__/getInitialStyles.spec.ts.snap

```diff
@@ -101,149 +101,136 @@ p-wordmark {
 .ssr {
   visibility: inherit;
 }
-p-accordion a,
-p-banner a,
-p-carousel a,
-p-checkbox-wrapper a,
-p-display a,
-p-flyout a,
-p-heading a,
-p-headline a,
-p-inline-notification a,
-p-modal a,
-p-multi-select a,
-p-pin-code a,
-p-popover a,
-p-radio-button-wrapper a,
-p-select-wrapper a,
-p-switch a,
-p-table a,
-p-tabs a,
-p-text a,
-p-text-field-wrapper a,
-p-text-list a,
-p-textarea-wrapper a {
+:is(
+    p-accordion,
+    p-banner,
+    p-carousel,
+    p-checkbox-wrapper,
+    p-display,
+    p-flyout,
+    p-heading,
+    p-headline,
+    p-inline-notification,
+    p-modal,
+    p-multi-select,
+    p-pin-code,
+    p-popover,
+    p-radio-button-wrapper,
+    p-select-wrapper,
+    p-switch,
+    p-table,
+    p-tabs,
+    p-text,
+    p-text-field-wrapper,
+    p-text-list,
+    p-textarea-wrapper
+  )
+  a {
   color: currentcolor !important;
+  transition: background var(--p-transition-duration, 0.24s) ease !important;
```

Comment:
Isn't this only relevant when `:hover` is supported?

Reply:
I think we should have the background transition defined in any case, since it does not harm but otherwise produces more CSS especially in the initial styles partial.